.github/workflows/cpaas.yml

```diff
@@ -95,9 +103,27 @@ jobs:
         vm_type:
           - rhel
           - rhel-sap
-            # Re-enable once s390x images exist for integration test
-            # components
-            #- rhel-s390x
+    with:
+      vm_type: ${{ matrix.vm_type }}
+      collector-tag: ${{ needs.init.outputs.collector-tag }}-cpaas
+      offline-mode: true
+      job-tag: cpaas
+    needs:
+    - init
+    - build-collector-full
+    secrets: inherit
+
+  multiarch-integration-tests:
```

Comment:
The idea here was to put the multi-arch tests behind an additional label to prevent the failures from breaking CI in master (looking at the `if` clause, this will still run on master so I'll fix that) 

we can certainly combine them again once we have fixes in place